tdrs-backend/tdpservice/urls.py

```diff
@@ -31,3 +35,24 @@
     path("v1/", include(urlpatterns)),
     path("admin/", admin.site.urls),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+
+
+schema_view = get_schema_view(
+    openapi.Info(
+        title="TDP API",
+        default_version='v1',
+        description="TANF Data Portal API documentation",
+        terms_of_service="TODO",
+        contact=openapi.Contact(email="TODO"),
```

Comment:
@carltonsmith I wasn't sure what to put here for `terms_of_service` or `contact` but both get exposed in the API documentation UI

Reply:
@jtwillis92 Is it possible to implement if we don't have a terms of service yet? Also, do you know what the email does? Does this email out anything, or just publish the email to the docs?

Reply:
Yeah those arguments are optional, they just control these links in the UI(s):
![2021-02-26_14-32](https://user-images.githubusercontent.com/22626085/109346289-78ce7080-783f-11eb-97d3-6eadcd9dcd65.png)
![2021-02-26_14-31](https://user-images.githubusercontent.com/22626085/109346291-79670700-783f-11eb-82e6-b5bb0886a071.png)
Without defining anything they are just not present in the UI / OpenAPI spec.